pkg/router/controller/host_admitter.go

```diff
@@ -134,7 +133,7 @@ func (p *HostAdmitter) HandleRoute(eventType watch.EventType, route *routev1.Rou
 	}
 
 	if err := p.admitter(route); err != nil {
-		glog.V(4).Infof("Route %s not admitted: %s", routeNameKey(route), err.Error())
+		log.V(4).Info("route not admitted: %s", "routeNameKey", routeNameKey(route), "error", err.Error())
```

Comment:
`routeNameKey(route)` is the stringified namespaced name, and I don't think a phrase like "name key" is commonly used outside this code, so how about either using `"routeNamespacedName", routeNameKey(route)` or `"namespace", route.Namespace, "name", route.Name`?

Reply:
fixed